west.yml

```diff
@@ -212,6 +212,9 @@ manifest:
       path: modules/crypto/tinycrypt
       groups:
         - crypto
+    - name: littlefs
+      path: modules/fs/littlefs
+      revision: pull/7/head
```

Comment:
Sorry I didn't notice this before but you need to modify the existing entry for littlefs in this manifest. Only need to change the "revision" line on that entry to point to your PR in that module. What you did here was add another entry for littlefs so now CI is breaking on this